src/jdk.crypto.cryptoki/share/classes/sun/security/pkcs11/P11ECKeyFactory.java

```diff
@@ -294,7 +300,14 @@ <T extends KeySpec> T implGetPublicKeySpec(P11Key key, Class<T> keySpec,
             try {
                 token.p11.C_GetAttributeValue(session[0].id(), keyID, attributes);
                 ECParameterSpec params = decodeParameters(attributes[1].getByteArray());
-                ECPoint point = decodePoint(attributes[0].getByteArray(), params.getCurve());
+                ECPoint point;
+
+                // If not using X963 encoding then extract the curve name from the octet string
+                if (token.config.getUseEcX963Encoding()) {
```

Comment:
This is a backport of https://github.com/openjdk/jdk/commit/55f5542c. The identical change should be backported, including the Oracle copyright update. There shouldn't be any IBM copyright added to this file.

Reply:
I've updated the file to be identical to the openjdk change, but it looks like removing the IBM copyright has caused the copyright check to fail.

Reply:
>  removing the IBM copyright has caused the copyright check to fail.

That's ok. It's up to the reviewer to determine if the failure is valid or not.